authorizer-app-backend/src/main/java/org/radarbase/authorizer/service/OuraAuthorizationService.kt

```diff
@@ -51,4 +64,17 @@ class OuraAuthorizationService(
             return false
         }
     }
+
+    private fun getExternalId(accessToken: String): String {
+        val ouraUserUri = UriBuilder.fromUri(OURA_USER_ID_ENDPOINT).queryParam("access_token", accessToken).build().toString()
+        val userReq = Request.Builder().apply {
+            url(ouraUserUri)
+        }.build()
+        val userIdObj: OuraAuthUserId = httpClient.requestJson(userReq, oauthUserReader)
```

Comment:
I think we can also add additional response/error handling here in case the request is unsuccessful.

Reply:
not sure how to do error handling after calling `requestJson`. Other option would be to execute request using this `client.newCall(request).execute()` and handle error like in `GarminSourceAuthorizationService.kt`. Happy to do it unless there is a better way to do the same thing

Reply:
Yes can we just switch to the implementation similar to `GarminSourceAuthorizationService`. Because in case in the future when the API gets deprecated or there are issues with this endpoint, we can handle the error. (Since this is using v1 of the API)